src/js/filter.js

```diff
@@ -139,7 +138,7 @@ class Filter {
           <div id='search-icon'>
             <img src='images/search-icon.svg' alt='search icon'></img>
           </div>
-          <input type="text" class="search-input" value="${DOMPurify.sanitize(this.searchOptions.initialSearch).replace(/\"/g, '')}" id="search" placeholder="Search sites or needs..."></input>
+          <input type="text" class="search-input" value="${this.searchOptions.initialSearch.replace(/\"/g, '')}" id="search" placeholder="Search sites or needs..."></input>
```

Comment:
Can you elaborate a bit on this change? It's not immediately obvious to me why the sanitize call is no longer necessary.

Reply:
Well the bug was really that html was being rendered at all, not that it was being rendered unsafely. By removing any quotes we can assure that any value passed in won't get misinterpreted as html and rendered at all, so its no longer necessary to sanitize the html. 

I think dompurify would be good if we wanted to safely render user-provided html.